model/userDAO.js

```diff
@@ -16,4 +17,17 @@ userSchema.methods.validPassword = function(password) {
 
 var User = mongoose.model('User', userSchema);
 
-module.exports = User;
+//convenience data access methods
+var userDAO = {
+  User: User,
```

Comment:
might want to just make the userDAO an object that inherits the prototype of User